### PR TITLE
deal with n_controllers=1

### PR DESCRIPTION
**Pull request by tomerfqm**

### CODE REVIEW

**Review by yomach — COMMENTED** [review]
Add to changelog

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Changed
 
+- simulator_tools.create_simulator_controller_connections - now deals with the case of 1 controller.
```
Comment (yomach) [inline CHANGELOG.md:10]: I would put it under fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -86,187 +87,4 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - config builder MeasurePulse now accepts IntegrationWeights object as well
 - config builder Element now accepts ControlPulse and MeasurePulse
 - config builder Parameter class now support basic algebra (+, -, /, *, **)
-- A new tool for discriminating two states in the IQ blob under `analysis.discriminator`
```
Comment (yomach) [inline CHANGELOG.md:89]: Why did you delete everything here?

File: CHANGELOG.md [hunk 3]
```diff
@@ -86,187 +87,4 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
...
-[0.3.3]: https://github.com/qua-platform/py-qua-tools/compare/v0.3.2...v0.3.3
-[0.3.2]: https://github.com/qua-platform/py-qua-tools/compare/v0.3.1...v0.3.2
-[0.3.1]: https://github.com/qua-platform/py-qua-tools/compare/v0.3.0...v0.3.1
-[0.3.0]: https://github.com/qua-platform/py-qua-tools/compare/v0.2.2...v0.3.0
-[0.2.2]: https://github.com/qua-platform/py-qua-tools/compare/v0.2.1...v0.2.2
-[0.2.1]: https://github.com/qua-platform/py-qua-tools/compare/v0.2.0...v0.2.1
-[0.2.0]: https://github.com/qua-platform/py-qua-tools/compare/v0.1.1...v0.2.0
-[0.1.1]: https://github.com/qua-platform/py-qua-tools/compare/v0.1.0...v0.1.1
-[0.1.0]: https://github.com/qua-platform/py-qua-tools/releases/tag/v0.1.0
+- A new tool f
```
Comment (yomach) [inline CHANGELOG.md:90]: ?

**Review by yomach — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (tomerfqm) [reply CHANGELOG.md:90]: wtf..

File: CHANGELOG.md (hunk 2 above)
Comment (yomach) [reply CHANGELOG.md:89]: that is why we have code reviews even for stupid stuff :)

**Review by TheoLaudatQM — APPROVED** [review]

**Review by yomach — APPROVED** [review]

**Pull request merged**